en/sensor_bus/i2c.md

```diff
@@ -1 +1,11 @@
-# I2C Bus Overview
\ No newline at end of file
+# I2C Bus Overview
+
+I2C is a packet-switched serial communication protocol that allows multiple master devices to connect to multiple slave devices using only 2 wires per connection. It is intended for attaching lower-speed peripheral ICs to processors and microcontrollers in short-distance, intra-board communication. 
+Pixhawk/PX4 support it for connecting off board components that require greater data rates than provided by a strict serial UART: e.g. rangefinders.
```

Comment:
Is this the only reason we choose I2C - because of the higher data rates than UART? Or are some hardwares likely only to be I2C compatible, so we'd choose in that case in order to be able to talk to this sort of hardware.

Reply:
Thanks, I'll update